tests/functional/aws-node-sdk/test/object/deleteMpu.js

```diff
@@ -2,28 +2,48 @@ import assert from 'assert';
 
 import withV4 from '../support/withV4';
 import BucketUtility from '../../lib/utility/bucket-util';
+import config from '../../../../../lib/Config';
 
 const bucket = 'functestabortmultipart';
 const key = 'key';
 
 const itSkipIfAWS = process.env.AWS_ON_AIR ? it.skip : it;
 
+const westLocation = config.locationConstraints ? 'scality-us-west-1'
+: 'us-west-1';
+const eastLocation = config.locationConstraints ? 'aws-us-east-1' : 'us-east-1';
+const describeSkipIfOldConfig = config.locationConstraints ? describe :
+describe.skip;
+
+// Why are we skipping error 404 if old config?
+// AWS returns 404 - NoSuchUpload in us-east-1. This behavior
+// can be toggled to be compatible with AWS by enabling
+// usEastBehavior in the config.
```

Comment:
If you look at the `confLocations` array, I tested with both us-east and us-west behavior.
This is just a note explaining why I use `describeSkipIfOldConfig` for us-east behavior.
I skip this test because I can not enabling usEastBehavior in the config.
Is it clearer?